.gitignore

```diff
@@ -1,23 +1,163 @@
-*.*~
-*.pyc
-vendor/*
-composer.phar
-*~$
-*\.log.*
-**/*.egg-info/*
-**/build/*
-**/dist/*
-*~
-/airtime_mvc/tests/test_results.xml
-/tests/results.html
-/tests/*.jar
-VERSION
-airtime_mvc/tests/log/*.log
-.vagrant/
+## Custom .gitignore
+################################################################################
+
 .DS_Store
 .idea/
-/docs/_site/*
-/docs/.jekyll-cache/*
-/docs/.gems/*
-Gemfile.lock
-api.log
+.vagrant/
+*.log
+*~
+VERSION
+
+## Github Composer .gitignore
+## See https://github.com/github/gitignore/blob/master/Composer.gitignore
+################################################################################
+
+composer.phar
+/vendor/
+
+# Commit your application's lock file https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control
+# You may choose to ignore a library lock file http://getcomposer.org/doc/02-libraries.md#lock-file
+# composer.lock
+
+## Github Python .gitignore
+## See https://github.com/github/gitignore/blob/master/Python.gitignore
+################################################################################
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.nox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.py,cover
+.hypothesis/
+.pytest_cache/
+cover/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+
+# Flask stuff:
+instance/
+.webassets-cache
+
+# Scrapy stuff:
+.scrapy
+
+# Sphinx documentation
+docs/_build/
```

Comment:
Should this be moved to `docs/.gitignore`?

Reply:
I copy pasted this whole section from the Github file, so I didn't want to touch it. I would leave it as is, but I am ok with moving it in the docs/.gitignore

Reply:
I commented it out.

Reply:
I think you can remove it, just needs to be in the docs one

Reply:
But we don't use sphinx documentation right ?

Reply:
Good point, not anymore. So actually can be removed completely